bom/build.gradle

```diff
@@ -12,7 +12,12 @@ publishing {
 
       pom.withXml {
         // Generate bom using subprojects
-        def internalProjects = [project.name, 'grpc-gae-interop-testing-jdk8', 'grpc-compiler']
+        def internalProjects = [
+          project.name,
+          'grpc-compiler',
+          'grpc-binder',
+          'grpc-gae-interop-testing-jdk8',
```

Comment:
Now that you switched the order of 'grpc-gae-interop-testing-jdk8' and 'grpc-compiler', why not insert 'grpc-binder' in dictionary order?

Reply:
Doh. Because I removed it during some testing and added it back in incorrectly.